### PR TITLE
Update ITEM_en.tsv

### PR DESCRIPTION
**Pull request by emailboxu**
Minor grammatical changes.
Removed some incorrect alternate translations. 
Few updates to 2H Sword strings .
Rod string changes.
Changed some instances of "Rod of X" to "X Rod" where appropriate (Lack of article "의" in these cases)
Changed some instances of "Rod" to "Staff" ("로드" vs "지팡이", latter could also be read as "Cane" but "Staff" seems more appropriate). 
Changed some instances of "Rod" to "Weapon" where unspecified in original translation ("로드" vs "장비", latter could also be read as "Equipment" but "Weapon" seems more suitable)
Bow string changes/polishing (Some major changes to descriptions; ie, 297, 300, 326, 328, 334, 345, 358, 360).

Edit: More fixes.

### CODE REVIEW

File: ITEM_en.tsv [hunk 1]
```diff
@@ -94,15 +94,15 @@ ITEM_20150317_000104	If you're unsatisfied with one type of Falchion, this is th
...
 ITEM_20150317_000109	Fortis
 ITEM_20150317_000110	Unpopular during the Great Plant Cataclysm, the resurgence of strong defense monsters is proving its value.
 ITEM_20150317_000111	Chapparition Cutter
 ITEM_20150317_000112	According to a theory, this sword is also used in battles amongst the Demons.
 ITEM_20150317_000113	Deathweaver Cutter
-ITEM_20150317_000114	Thou, wielder of this blade, shalt become the deathbringer. / Now that you wield this blade, you must become the deathbringer.
+ITEM_20150317_000114	Thou, wielder of this blade, shalt become the deathbringer. / Now that you wield this blade, you will become the deathbringer.
 ITEM_20150317_000115	Chapparition Sword
-ITEM_20150317_000116	It is also the indirect evidence that there were disputes between Demons.
+ITEM_20150317_000116	It is the indirect evidence that there were disputes between Demons.
```
Comment (emailboxu) [inline ITEM_en.tsv]: This is the indirect evidence __

File: ITEM_en.tsv [hunk 2]
```diff
@@ -157,7 +157,7 @@ ITEM_20150317_000171	Swordsmen who seeks raw power in their weapon frequently ch
 ITEM_20150317_000172	Kindzal
 ITEM_20150317_000173	Those who likes to temper their weapons craft this sword for their own use.
 ITEM_20150317_000174	Imperni Two-Handed Sword
-ITEM_20150317_000175	If you got this sword using the black market, then It is an evidence that was obtained certain trust.
+ITEM_20150317_000175	If you got this sword using the black market, then it is evidence that you have obtained a certain level of reliability.
```
Comment (emailboxu) [inline ITEM_en.tsv]: "If you got this sword through the black market, it is evidence..."

File: ITEM_en.tsv [hunk 3]
```diff
@@ -283,21 +283,21 @@ ITEM_20150317_000309	Widely crafted by people across the kingdom and its territo
...
-ITEM_20150317_000323	First ever recorded case of applying magic in the bowcrafting process. Since then it has found its own market.
+ITEM_20150317_000323	The first ever recorded case of applying magic in the bowcrafting process. Since then it has found its own market.
 ITEM_20150317_000324	Karman
 ITEM_20150317_000325	A bow modified for more power by archers confident in their arm strength.
-ITEM_20150317_000328	Army Light Bow
+ITEM_20150317_000328	Soldier's Light Bow
 ITEM_20150317_000329	This bow has been crafted to meet the minimum requirements of the Army.
 ITEM_20150317_000330	Mayor's Short Bow
-ITEM_20150317_000331	It shows the signs of tying and untying the string everyday. Despite its age, the bow has been taken a good care.
+ITEM_20150317_000331	It shows signs of the effort put into unstringing and restringing the bow every day. Despite its age, the bow has been taken a good care.
```
Comment (emailboxu) [inline ITEM_en.tsv]: "Despite its age, the bow is in good condition."

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**